.ci/atime/tests.R

```diff
@@ -208,5 +208,20 @@ test.list <- atime::atime_test_list(
     Slow = "fd24a3105953f7785ea7414678ed8e04524e6955", # Parent of the merge commit (https://github.com/Rdatatable/data.table/commit/ed72e398df76a0fcfd134a4ad92356690e4210ea) of the PR (https://github.com/Rdatatable/data.table/pull/5054) that fixes the issue
     Fast = "ed72e398df76a0fcfd134a4ad92356690e4210ea"), # Merge commit of the PR (https://github.com/Rdatatable/data.table/pull/5054) that fixes the issue
 
+# Improvement discussed in and brought by https://github.com/Rdatatable/data.table/pull/4386
+"forderv improved in #4386" = atime::atime_test(
+  N = 10^seq(3, 8), # 1e9 exceeds the runner's memory (process gets killed)
+  setup = {
+    options(datatable.forder.auto.index = TRUE, datatable.forder.reuse.sorting = TRUE)
+    dt <- data.table(index = sample(N), values = sample(N))
```

Comment:
please add set.seed(1) for reproducibility in setup, whenever you use random generation

Reply:
Done! (and noted)